doc/lua_api.txt

```diff
@@ -5616,6 +5621,8 @@ Global tables
     * Map of registered tool definitions, indexed by name
 * `minetest.registered_entities`
     * Map of registered entity prototypes, indexed by name
+    * Values in this table may be modified directly.
+      Note: the new values will only affect entities spawned afterwards
```

Comment:
This is not true - it uses metatables, so the entity table will immediately change for all entities. However, initial properties are only read when entities are spawned

```suggestion
      Note: changes to initial properties will only affect entities spawned afterwards,
      as they are only read when spawning.
```